html_xblock/html.py

```diff
@@ -59,7 +59,7 @@ def resource_string(path):
     @XBlock.supports('multi_device')
     def student_view(self, context=None):  # pylint: disable=unused-argument
         """
-        Return a fragment that contains the html for the student view
+        Return a fragment that contains the html for the student view.
         """
         frag = Fragment()
         frag.content = xblock_loader.render_template('static/html/lms.html', {'self': self})
```

Comment:
@taniwha, the below warning message is printed when running the tests. Can we fix this and the 2 other occurrences?

```
UserWarning: ResourceLoader.render_template has been deprecated in favor of ResourceLoader.render_django_template
```